README.md

```diff
@@ -32,6 +32,27 @@ The different alerting rules in this repository are:
 
 * [**Alert Rule QuotaExceeded**](https://gitlab.cee.redhat.com/rhtap/docs/sop/-/blob/main/o11y/alert-rule-QuotaExceeded.md)
 
+### SLO Alerts
+
+SLO (Service Level Objective) alert rules are rules defined to monitor and alert when a service or system is not meeting its specified service level objectives.
+
+#### **When to apply:**
```

Comment:
This header is a bit confusing as a reader because I must read on to understand what may need to be applied. This also makes it seem like the SLO label is optional when it's more of a requirement.

Reply:
Updated the headers